### DISCUSSION TITLE
min browser with version ungoogled chrome?

### DISCUSSION THREAD

**Discussion by ghost** [minbrowser/min#2159] [Q&A] [answered]
Hi everyone.

I would like to know if you have thought about creating a version of min browser based on ungoogled chrome.

#### why?
[ungoogled-chromium](https://github.com/ungoogled-software/ungoogled-chromium) is a free and open-source variant of the Chromium web browser that removes all Google-specific web services. It achieves this with a series of patches applied to the Chromium codebase during the compilation process. The result is functionally similar to regular Chromium.

#### question/feedback
*what do you all think of this idea?*

**Comment by PalmerAL** [comment, marked as answer]
Answered in https://github.com/minbrowser/min/issues/1922.

**Reply by ghost** [reply]
Hi PalmerAL.

> Answered in https://github.com/minbrowser/min/issues/1922.

thanks!
